### DISCUSSION TITLE
Hot reloading

### DISCUSSION THREAD

**Discussion by Kiwa21** [bluenviron/mediamtx#263] [Questions]
## Which version are you using?

v0.12

## Which operating system are you using?

Linux

## Question

Hello aler9 ! First of all, thanks for this great piece of software :) 
You said in the readme : Reload the configuration without disconnecting existing clients (hot reloading)
How does one hot reload the configuration?

**Comment by aler9** [comment]
Hello, just edit the configuration file and save it.
The software will detect that the configuration file was edited and will reload it.

This is the only solution i've found to be compatible with all supported OSs, since Windows doesn't support signals.

[a comment by an automated account is not shown]
